### DISCUSSION TITLE
I want to create a pull request but I'm blocked

### DISCUSSION THREAD

**Discussion by XDavidT** [guerzon/vaultwarden#14] [General]
How I can push a new branch and open PR? I get an error trying to push:
`Permission to guerzon/vaultwarden.git denied `

**Comment by guerzon** [comment]
Hello, you need to fork the repository, work on your branch, then create a pull request. See [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
